### PR TITLE
Port fix model without anyservers

### PR DESCRIPTION
**Pull request by jshum2479**
This fixes a problem where the WDT model has no servers defined but with configured cluster defined.  This should causes a validation error.

### CODE REVIEW

File: operator/src/main/resources/scripts/introspectDomain.py [hunk 1]
```diff
@@ -493,7 +493,7 @@ def validateNonDynamicClusterReferencedByAtLeastOneServer(self, cluster):
     for server in self.env.getDomain().getServers():
       if self.env.getClusterOrNone(server) is cluster:
         return
-    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.")
+    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.  You must have managed servers defined and belong to this cluster")
```
Comment (tbarnes-us) [inline operator/src/main/resources/scripts/introspectDomain.py:496]: and belong --> that belong

Comment (tbarnes-us) [reply operator/src/main/resources/scripts/introspectDomain.py:496]: cluster --> cluster.

File: operator/src/main/resources/scripts/introspectDomain.py [hunk 2]
```diff
@@ -493,7 +493,7 @@ def validateNonDynamicClusterReferencedByAtLeastOneServer(self, cluster):
     for server in self.env.getDomain().getServers():
       if self.env.getClusterOrNone(server) is cluster:
         return
-    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.")
+    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.  You must have managed servers defined that belong to this cluster")
```
Comment (ankedia) [inline operator/src/main/resources/scripts/introspectDomain.py:496]: Should there be a period after "this cluster" i.e. "this cluster." ?

Comment (jshum2479) [reply operator/src/main/resources/scripts/introspectDomain.py]: fixed

File: operator/src/main/resources/scripts/introspectDomain.py (hunk 1 above)
Comment (jshum2479) [reply operator/src/main/resources/scripts/introspectDomain.py]: fixed

**Review by ankedia — APPROVED** [review]

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**